diff --git a/config.def.h b/config.def.h
index 4ce731d..db85d54 100644
--- a/config.def.h
+++ b/config.def.h
@@ -19,12 +19,13 @@ static const char col_gray2[]       = "#444444";
 static const char col_gray3[]       = "#bbbbbb";
 static const char col_gray4[]       = "#eeeeee";
 static const char col_cyan[]        = "#005577";
+static const char col_orange[]      = "#ff4f00";
 static const unsigned int baralpha = 0xd0;
 static const unsigned int borderalpha = OPAQUE;
 static const char *colors[][3]      = {
 	/*               fg         bg         border   */
 	[SchemeNorm] = { col_gray3, col_gray1, col_gray2 },
-	[SchemeSel]  = { col_gray4, col_cyan,  col_cyan  },
+	[SchemeSel]  = { col_gray4, col_orange,col_orange  },
 };
 static const unsigned int alphas[][3]      = {
     /*               fg      bg        border*/
@@ -92,7 +93,7 @@ static char dmpx[3] = "10";
 static char dmpy[3] = "10";
 static char dmpz[5] = "1900"; 
 			//
-static const char *dmenucmd[] = { "dmenu_run", "-m", dmenumon, "-fn", dmenufont, "-nb", col_gray1, "-nf", col_gray3, "-sb", col_cyan, "-sf", col_gray4, "-x", dmpx , "-y", dmpy, "-z", dmpz, NULL };
+static const char *dmenucmd[] = { "dmenu_run", "-m", dmenumon, "-fn", dmenufont, "-nb", col_gray1, "-nf", col_gray3, "-sb", col_orange, "-sf", col_gray4, "-x", dmpx , "-y", dmpy, "-z", dmpz, NULL };
 static const char *termcmd[]  = { "st", NULL };
 
 static const char *brightnessUp[] = { "backlight", "incr","10", NULL };
