diff --git a/config.def.h b/config.def.h
index b80f32b..c0dd5c5 100644
--- a/config.def.h
+++ b/config.def.h
@@ -43,9 +43,11 @@ static const Rule rules[] = {
 	 *	WM_CLASS(STRING) = instance, class
 	 *	WM_NAME(STRING) = title
 	 */
-	/* class      instance    title       tags mask     isfloating   monitor */
-	{ "Gimp",     NULL,       NULL,       0,            1,           -1 },
-	{ "Firefox",  NULL,       NULL,       1 << 8,       0,           -1 },
+  /* class      instance    title                 tags mask     isfloating   monitor */
+  { "Gimp",     NULL,       NULL,                 0,            1,           -1 },
+  { "Firefox",  NULL,       NULL,                 1 << 8,       0,           -1 },
+  { "firefox",  "Toolkit",  "Picture-in-Picture", -1,           1,           -1 },
+  { "mpv",      NULL,       NULL,                 0,            1,           -1 }
 };
 
 /* layout(s) */
